experiment/textcov.py

```diff
@@ -30,6 +30,20 @@ def demangle(data: str) -> str:
   return subprocess.check_output(['c++filt'], input=data, encoding='utf-8')
 
 
+def _discard_fuzz_target_lines(covreport_content: str) -> str:
+  """Removes fuzz target lines from the coverage report."""
+  # When comparing project code coverage contributed by fuzz targets, it's
+  # fairer to only consider lines in the project and not the code of targets.
+  # Assumption 1: llvm-cov separates lines from different files with an empty
```

Comment:
This seems like a reasonable assumption based on skimming https://github.com/llvm/llvm-project/blob/662d821d44420566cd0bb649f30d7689562bdb88/llvm/tools/llvm-cov/SourceCoverageViewText.cpp#L30 and https://github.com/llvm/llvm-project/blob/662d821d44420566cd0bb649f30d7689562bdb88/llvm/tools/llvm-cov/CodeCoverage.cpp#L671

But we need to test this  on more benchmarks to make sure there are no valid instances of \n\n elsewhere.

Reply:
Thanks for showing the code evidence!

Let me merge this now to unblock another PR, then I can run another full experiment an check for anomalies.